### ISSUE TITLE
API Guide: Native API, Create Dataset endpoint uses out of date sample file, fails when used.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#9513]
In the Native API guide
https://guides.dataverse.org/en/5.13/api/native-api.html#create-a-dataset-in-a-dataverse-collection

This file no longer matches the metadata blocks and so needs to be updated.
https://guides.dataverse.org/en/5.13/_downloads/4e04c8120d51efab20e480c6427f139c/dataset-create-new-all-default-fields.json

productionPlace needs to be multiple: true and bounding box entries are now validated so values need correction.

**Comment by pdurbin** [comment]
Yes. In addition, we should add `FeedbackApiIT` to `tests/integration-tests.txt` because it exercises `dataset-create-new-all-default-fields.json`.
